suite-native/device-manager/src/components/DeviceList.tsx

```diff
@@ -51,6 +60,27 @@ const buttonWrapperStyle = prepareNativeStyle(utils => ({
     paddingTop: utils.spacings.small,
 }));
 
+const ConnectButtonComponent = ({ isDividerVisible, onPress }: ConnectButtonComponentProps) => {
```

Comment:
Everything is a component :D
```suggestion
const ConnectButton = ({ isDividerVisible, onPress }: ConnectButtonProps) => {
```

Reply:
Yeah I know, but it this felt better than just ConnectButton, since it might be more than just button. I couldnt come with better name than this one :)

Reply:
but if it feels good for you, I'll change it :)

Reply:
e6ffd60283

Reply:
Now it is only a button so it should be called like this IMO. When we add something to the component, then it should be renamed.